oak/proto/policy.proto

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.policy;
+
+// Labels represents a policy associated with a message or a Node.
```

Comment:
I'm struggling to get the equivalence between "labels" and "policy" throughout the PR; "policy" makes me think of something different and higher level than just a collection of tags.

Reply:
Thanks, I will clarify in the documentation, but basically in my mind policy == set of secrecy tags. Do you think that is insufficient? Could you give an example of something you were thinking that cannot be represented in that way?

Reply:
Thanks, I will clarify in the documentation, but basically in my mind policy == set of secrecy tags. Do you think that is insufficient? Could you give an example of something you were thinking that cannot be represented in that way?

Reply:
If it is the case that "policy" == "set of secrecy tags" (which seems plausible atm), then we need to sort out the duplicate terminology: it's also the case that "secrecy label" == "set of secrecy tags" (as per the Flume paper) and referring to the same concept with different words is definitely adding to my confusion!  (Personally, I'd pick "label" but up to you.)

BTW, where do Flume-like integrity tags fit in with this concept of "policy"?

Reply:
I did think about adding another message

```
message Label {
  repeated Tag tags = 1;
}
```

but I'm not sure it would actually help clarify things.

The way I think about it, a policy is a set of secrecy tags that can be explained to users, but when data gets processed and mixed with data with other policies and "unnamed" tags, then the resulting label is not any more a policy. So policies are a subset of all labels, and also have the property that they enter the system over gRPC.

On the other hand, integrity labels would presumably be something that is attached to metadata sent from the server to the client, which the client may inspect to figure out the provenance of the data it's receiving, if it cares about it.

Does this make sense, or is it more confusing than before?

P.S.: I will write down some of these things in README / CONCEPTS in a separate PR.